streamz/dataframe/utils.py

```diff
@@ -0,0 +1,34 @@
+def is_dataframe_like(df):
```

Comment:
I think we should copy - or import to prevent drift - the test `dask.dataframe.tests.test_utils_dataframe::test_is_dataframe_like` (which checks all three functions). Dask is part of the test built, or could skip when dask not present; in any case, don't want to duplicate dask code (function or test) if you can think of a way to avoid it.

Reply:
Added separate tests for utils. but imports dask tests for methods borrowed from dask.datafram.utils .

Currently I can't think a way to avoid this code duplication. May be we can list dask in the project requirements for Streamz DataFrame users?.

Reply:
No, we probably shouldn't do that, it should be possible to use streamz-cudf or streamz-pandas without dask